Gemfile

```diff
@@ -3,6 +3,13 @@ gemspec :name => "jekyll"
 
 gem "rake", "~> 12.0"
 
+if RUBY_VERSION >= '2.4'
+  gem "json", "~> 2.0"
+  gem "pygments.rb", "~> 1.1"
+else
+  gem "pygments.rb", "~> 0.6.0" unless RUBY_ENGINE == "jruby"
```

Comment:
we can.. but I wasn't sure if the change would affect Jekyll internally in any way..
I pushed a new commit to use just one version of `pygments.rb`

From the logs, it looks like we just need to change our assertions to match the new pygments syntax. But I don't want to undertake that task.

So, you're free to close this PR or assign someone to directly modify this branch.